src/app/analytics/analytics.factory.js

```diff
@@ -509,6 +509,8 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
     },
     transactionEvent: function (purchaseData) {
       try {
+        // The value of whether or not user is covering credit card fees for the transaction
+        const coverFees = localStorage.getItem('coverFees')
```

Comment:
Is this triggered before or after [this line](https://github.com/CruGlobal/give-web/blob/master/src/app/checkout/step-3/step-3.component.js#L146)?

Reply:
Also, see `retrieveCoverFeeDecision`. This should get the data the same way.

Reply:
Ah looks like it's called after 🙁 I thought I was getting a value but testing it again now I'm getting ```null```.  Do you see another way to approach it? Maybe setting a temp sessionStorage with the value of ```coverFees``` and then deleting it after the transaction event?

Reply:
2 options:

1. Move the deletion of the variable
2. Make another event